extensions/wikia/MyHome/MyHome.class.php

```diff
@@ -181,14 +181,18 @@ public static function getInitialMainPage(Title &$title) {
 		//this is not used for Corporate Sites where Wikia Visualization is enabled
 		if( $wgUser->isLoggedIn() && empty($wgEnableWikiaHomePageExt) ) {
 			$value = $wgUser->getGlobalPreference(UserPreferencesV2::LANDING_PAGE_PROP_NAME);
-			switch($value) {
-				case UserPreferencesV2::LANDING_PAGE_WIKI_ACTIVITY:
-					$title = SpecialPage::getTitleFor('WikiActivity');
-					break;
-				case UserPreferencesV2::LANDING_PAGE_RECENT_CHANGES:
-					$title = SpecialPage::getTitleFor('RecentChanges');
-					break;
+			if ( $value == UserPreferencesV2::LANDING_PAGE_WIKI_ACTIVITY ) {
```

Comment:
It's pretty minor thing here but in my opinion is cleaner what's happening in `switch case` statement. And to avoid `if else` inside you can use ternary operator or create separate method to get proper title. But it's up to you :)